f/medical/fn_famInit.sqf

```diff
@@ -0,0 +1,68 @@
+// F3 FA Medical
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// ====================================================================================
+
+// INITIALIZE
+_unit = _this;
+
+// ====================================================================================
+
+// MEDICAL SCRIPTS
+// Start Health Loop for monitoring bleed/consciouness
+_unit spawn f_fnc_famLoop;
+
+// Make downed players draggable
+[player] remoteExec ["f_fnc_famAddDragAction", 0, player]; 
+
+// Handle damage appropriately for the system.
+_unit call f_fnc_famDamageHandler;
```

Comment:
Does every player have to add this EH to every other playable unit?
Shouldnt these things only be applied to where the unit is local?

Reply:
I definitely have a very inconsistent handling of AI currently. Playable units was used for early stages of testing so I could test on AI, but I think in the final version it makes sense to run none of this on AI.

Reply:
My point wasnt about AI. It was about where the EH should be added and where it should be run. Does it run whereever the EH was added or just where the unit is local? Does this get executed multiple times on multiple machines whenever someone takes damage? (I don't know a lot about EHs). If we only care about the player themself then we should probably use `player` as the argument here and also make sure that it only gets added once.

Reply:
Adjustments have been made so that everything in the init only runs when the unit is local to yourself. Which means AI will get the damage handler and be beefier on the off chance they get shot before a real human joins the slot. 

According to Chozo's damage handler, the handler _does_ run when a unit is damaged but is not local to you (not sure if this is all units or would just be units that were once local to you). But that damage handler has a quick exit if !local which will avoid running that business once the unit is no longer local.  It is my understanding that damage is handled locally otherwise, and even if the damage handler triggers for a non-local unit it would not update a remote unit unless otherwise remoteExeced?  

Not sure if what I've got will be the best way to handle it, so leaving this comment open if others have better information.